src/public/modules/forms/base/controllers/submit-form.client.controller.js

```diff
@@ -27,20 +27,55 @@ function SubmitFormController(
   // The form attribute of the FormData object contains the form fields, logic etc
   vm.myform = FormData.form
 
+  const isSpcpSgidForm = !!['SP', 'CP', 'SGID'].includes(vm.myform.authType)
+  const isMyInfoForm = !!(
+    !FormData.isTemplate && vm.myform.authType === 'MyInfo'
```

Comment:
as far as I can tell it shouldn't matter, but this was a pure refactor of current implementation, so i'll leave it for now (since it will be rewritten in react)